src/components/footnote-group/footnote-group.twig

```diff
@@ -0,0 +1,27 @@
+<div class="c-footnote-group{% if compact %} c-footnote-group__compact{% endif %}">
```

Comment:
Should this be a modifier? 🔍 👀 
```suggestion
<div class="c-footnote-group{% if compact %} c-footnote-group--compact{% endif %}">
```

Reply:
Oh, yep :+1: Good catch

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/8fb6e439a35a084016142a61903dba80cf4d24e6